react/components/FilterSidebar.js

```diff
@@ -110,7 +112,21 @@ const FilterSidebar = ({
     setFilterOperations([])
   }
 
+  const { push } = usePixel()
+
+  const pushClearEvent = () => {
+    push({
+      event: 'filterManipulation',
+      items: {
+        filterProductCategory: getCategoryFromObjs(searchQuery.products),
+        filterName: 'CleanFilters',
+        filterValue: true,
+      },
+    })
+  }
+
   const handleClearFilters = (key) => {
+    pushClearEvent()
```

Comment:
Same thing here, I think you should use the function you exported

Reply:
Same thing, changed using the pushPixelEvent function